<launch>
    <group ns="move_base_seq">
        <rosparam file="$(find p_agbot)/param/autonomous_navigation_param.yaml" />
    </group>

    <node pkg="p_agbot" type="autonomous_navigation.py" name="autonomous_navigation" output="screen" />
</launch>
